src/annotation/spline.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { Toolboxes } from "@/Toolboxes";
+import { Annotations } from "./index";
+
+let annotationsObject: Annotations;
+
+describe("Convert Splines to Paintbrush", () => {
+  beforeEach(() => {
+    annotationsObject = new Annotations();
+    annotationsObject.addAnnotation(Toolboxes.spline);
+  });
+
+  test("test add spline points", () => {
+    annotationsObject.addSplinePoint({ x: 2, y: 5 }, false);
+    const coordinates = annotationsObject.getSplineCoordinates();
+    expect(coordinates).toEqual([{ x: 2, y: 5 }]);
+  });
+
+  test("test convert spline to paintbrush", () => {
+    annotationsObject.addSplinePoint({ x: 2, y: 5 }, false);
+    annotationsObject.convertSplineToPaintbrush(0.5);
+    const brustrokesCord = annotationsObject.getBrushStrokeCoordinates();
+    expect(brustrokesCord).toEqual([{ x: 2, y: 5 }]);
```

Comment:
after the conversion you could also check whether or not `annotationsObject` contains a spline annotation.